<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FCanceledReservations</class>
 <widget class="QWidget" name="FCanceledReservations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Date from</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EDateEdit" name="deStart"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Date to</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EDateEdit" name="deEnd"/>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>196</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Search by date of</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbEntry">
     <property name="text">
      <string>Entry</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbDeparture">
     <property name="text">
      <string>Departure</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">edateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
